src/actions/auth.js

```diff
@@ -0,0 +1,62 @@
+import auth from 'panoptes-client/lib/auth'
+import store from 'react-native-simple-store'
+import { Actions, ActionConst } from 'react-native-router-flux'
+
+import { checkIsConnected, setState, setIsFetching } from '../actions/index'
+import { loadUserAvatar, syncUserStore } from '../actions/user'
+
+export function getAuthUser() {
+  return () => {
```

Comment:
Is there a reason this promise is written inside this extra function? I'm sure this is purposeful, I'm mainly curious what the reason is.